dev-tools/check_file_permissions.sh

```diff
@@ -6,7 +6,7 @@ files_with_wrong_permissions=$(
         ':!*.sh' \
         ':!php-cs-fixer' \
         ':!dev-tools/*.php' \
-    | grep -P "^100\d\d(1|3|5|7) " \
+    | grep -P "^100755 " \
```

Comment:
then you could also "simplify" `-P` to `-E`

Reply:
wouldn't `-G`  (the default, so we can drop the flag) fit best?

Reply:
ah, basic `grep "^100755 "` seems enough indeed

Reply:
sorry guys missed this improvement during merging :(